src/main/java/org/openRealmOfStars/utilities/repository/GameRepository.java

```diff
@@ -37,22 +37,9 @@
 public class GameRepository {
 
   /**
-   * Is GameRepository called for JUnit
-   */
-  private boolean runningJUnit;
-
-  /**
    * Default constructor used in actual game
    */
   public GameRepository() {
-    runningJUnit = false;
-  }
-
-  /**
-   * Enable loading save files under JUnit folder
-   */
-  public void enableJUnit() {
-    runningJUnit = true;
```

Comment:
I knew this was bad design but this was the only solution I came up that could be done to get actual full save game loading to be unit tested.